include/drjit-core/half.h

```diff
@@ -0,0 +1,119 @@
+/*
+    drjit-core/half.h -- minimal half precision number type
+
+    Copyright (c) 2021 Wenzel Jakob <wenzel.jakob@epfl.ch>
```

Comment:
Hm, it's weird to list me as copyright author given that you wrote this code. Why don't you remove it for now or put your name? Or we can change the copyright note to list the lab in a later commit.